src/store/modules/groups.js

```diff
@@ -69,7 +69,7 @@ export default {
         const data = await groups.save(group)
         commit('update', data)
         dispatch('currentGroup/update', data, { root: true })
-        router.push({ name: 'group', params: { groupId: group.id } })
+        router.go(-1)
```

Comment:
This does nothing if the user opens the edit link directly and saves the form, e.g. https://dev.karrot.world/#/group/6/edit

It does the wrong thing if the user opens this link from another website (e.g. a chat application) in the same tab. Then it would direct the user back to the chat application after saving the form.

As the browser `history.length` does not help us very much there (it only shows the number of pages the user has visited in the current tab, but we don't know if the pages before have been on karrot), it might make sense to introduce a state that indicates if the current page is the first karrot page the user is visiting.

It could be implemented as [vue-router navigation guard](https://router.vuejs.org/en/advanced/navigation-guards.html), checking if the `from` state is defined. For [redirects](https://router.vuejs.org/en/essentials/redirect-and-alias.html), navigation guards are only executed for the destination, so it's necessary to check if the `from` state is still as expected.